vNext/src/OrleansSQLUtils/OrleansSQLUtils.csproj

```diff
@@ -0,0 +1,56 @@
+﻿<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" />
+  <PropertyGroup Label="Configuration">
+    <DefineConstants>NETSTANDARD;NETSTANDARD_TODO</DefineConstants>
+  </PropertyGroup>
+  <PropertyGroup>
+    <TargetFramework>netstandard1.5</TargetFramework>
+    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
+    <PackageTargetFallback>$(PackageTargetFallback);portable-net45+win8+wpa81+wp8</PackageTargetFallback>
```

Comment:
you don't necessary need this. AzureUtils need this is because Windows.AzureStorage package need this. Couple of AzureStorage package's dependency lib doesn't support netstandard yet. So need this as a fallback to load those libs. Took me a long time nicely fix it. 

Not sure SQLUtils need this. try remove it and to see if it builds